.github/workflows/build-windows.yml

```
@@ -64,6 +64,19 @@ jobs:
           }
           Write-Host "Docker daemon OS: $os"
 
+      # --------------------------------------------------------
+      # 2c. Derive cache tag from branch name
+      # --------------------------------------------------------
+      - name: Derive cache tag
+        id: cache-tag
+        shell: powershell
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)